### ISSUE TITLE
BIWI cal_metric training_subjects are the same training data?

### ISSUE THREAD

**Issue by youngstu** [Doubiiu/CodeTalker#23]
BIWI cal_metric training_subjects are the same with training data. 
Maybe cal_metric training_subjects should be the same with test data.

https://github.com/Doubiiu/CodeTalker/blob/e687bbe64bb3553d5653041a9d80a62eae593ebf/main/cal_metric.py#L10

**Comment by Doubiiu** [comment]
Not true. It should be the same (please read the paper carefully to distinguish between BIWI-Test-A and BIWI-Test-B).

**Comment by youngstu** [comment]
> Not true. It should be the same (please read the paper carefully to distinguish between BIWI-Test-A and BIWI-Test-B).

Got it. Thx.
